Scenarios/StepDefinition.swift

```diff
@@ -27,8 +27,9 @@ open class StepDefinition: QuickSpec {
 
     return {
       guard let (args, definition) = stepDefinitions.lazy
-        .flatMap({ pattern, function in
-          pattern.firstMatch(in: description).map {
+        .flatMap({ args -> (StepArguments, StepDefinitionFunc)? in
```

Comment:
I was never quite happy with how this turned out, so I did a bit of a refactor: a2ce62d.